common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message.h

```diff
@@ -70,51 +70,71 @@ class message_com {
     }
 
     template <class T>
-    static std::shared_ptr<T> create_vs_message(ieee1905_1::CmduMessageTx &cmdu_tx, uint16_t id = 0)
+    static std::shared_ptr<T> add_intel_vs_data(ieee1905_1::CmduMessageTx &cmdu_tx, uint16_t id = 0)
```

Comment:
Also here: commit AutoGenerated in different  commit

Reply:
Same answer - lets see what @arnout has to say